code/pizzabot.js

```diff
@@ -1,7 +1,65 @@
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const margherita = "Margherita Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+const pizzaVegImg = document.getElementById("veg")
+const pizzaHawImg = document.getElementById("haw")
+const pizzaPepImg = document.getElementById("pep")
+const pizzaMarImg = document.getElementById("mar")
+
+
+console.log(vegetarian, hawaiian, pepperoni, pizzaPrice)
+
+document.getElementById("menu").innerHTML = `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian}, ${pepperoni}, and ${margherita}.`
+
+
+const calculateTotalCost = (orderQuantity) => {
```

Comment:
There is an arrow function convention which says that if we pass to our function only one argument, we can omit parentheses `()`, and rather write :

`const calculateTotalCost = orderQuantity => { ... }`

than

`const calculateTotalCost = orderQuantity => { ... }`

Important thing to notice - this convention should be applied ONLY when we pass one argument. If we would do this with multiple arguments, it would cause `SyntaxError` to occur